pub_sub/go/sdk/checkout/go.mod

```diff
@@ -1,15 +1,18 @@
 module dapr_example
 
-go 1.17
+go 1.18
+
+require github.com/dapr/go-sdk v1.4.0
 
 require (
-	github.com/dapr/go-sdk v1.2.0 // indirect
-	github.com/golang/protobuf v1.4.3 // indirect
+	github.com/dapr/dapr v1.6.1-0.20220223052122-c503710ab2d1 // indirect
```

Comment:
Do you need to pull in dapr v1.6.1 into this as a dependency? And if so, best move to v1.7.2